src/components/Card.js

```diff
@@ -58,11 +75,17 @@ export default class Card {
   #completeNewCard () {
     this.#card = this.#returnEmptyClone();
     this.#likeButton = this.#card.querySelector(".card__like-button");
+    this.#likeCounter = this.#card.querySelector(".card__like-count");
     this.#cardImage = this.#card.querySelector(".card__image");
     this.#cardCaption = this.#card.querySelector(".card__caption");
     this.#deleteButton = this.#card.querySelector(".card__delete");
     this.#fillMarkupWithData();
     this.#addEventListeners();
+    
+    if (this.#owner._id != "da6f0aaa844471b7fec4eea5") {
```

Comment:
`da6f0aaa844471b7fec4eea5`


It's a hardcode of `_id`


`_id` of the user should be received from the server only 1 time. You need to store it in a variable `userId` or in class `UserInfo` and use it when you create a card passing it to the constructor of `Card`